packages/clients/textLocator/README.md

```diff
@@ -0,0 +1,11 @@
+# POLAR client TextLocator
+
+## Content
+
+The TextLocator client offers usage of the TextLocator backend that allows searching for papers in a geospatial fashion; that is, selecting a region will produce a list of papers found regarding that region.
+
+Please see the CHANGELOG.md for all changes after the initial release.
```

Comment:
Copied it from DISH. 🤷 I guess it's nice since the API.md is a customer-facing document for both clients and this clarification may help readers from a less technical background.

Reply:
It seems like this information should then be added to the `API.md` for both clients - not to the `README.md`.